OpenGL/Math/Matrix2.cs

```diff
@@ -86,6 +86,45 @@ public static Matrix2 Identity
             }
         }
 
+        public float this[int column, int row]
+        {
+            get
+            {
+                if (row == 0)
+                {
+                    if (column == 0) return row1.X;
+                    if (column == 1) return row1.Y;
+                }
+                if (row == 1)
+                {
+                    if (column == 0) return row2.X;
+                    if (column == 1) return row2.Y;
+                }
+                if (column > 1 || column < 0 || row > 1 || row < 0)
+                    throw new ArgumentOutOfRangeException("column or row is out of the matrix bounds.");
+                else return -1;
+            }
+            set
+            {
+                bool set = false;
```

Comment:
Same comment as on other Matrix classes, set is unnecessary and move exception up here.